### PR TITLE
Jli/rust pll instructions

### PR DESCRIPTION
**Pull request by throwbackjams**
Main components:
- check for ATAs and create instructions for their creation if needed
- check for seat and create instruction for claim seat if needed
- wrap above with create limit order and post only order, respectively, to create encompassing ix bundle
-  use encompassing ix bundle (appropriate for new users) in the default SDK send methods

### CODE REVIEW

File: rust/phoenix-sdk/src/sdk_client.rs [hunk 1]
```diff
@@ -522,4 +524,118 @@ impl SDKClient {
...
+            create_ata_ix_if_needed(&self.client, &self.trader, &self.trader, &self.base_mint)
+                .await;
+
+        let quote_ata_instruction_option =
+            create_ata_ix_if_needed(&self.client, &self.trader, &self.trader, &self.quote_mint)
+                .await;
+
+        let claim_seat_ix_option =
+            create_claim_seat_ix_if_needed(&self.client, &self.active_market_key, &self.trader)
+                .await;
```
Comment (jarry-xiao) [inline rust/phoenix-sdk/src/sdk_client.rs:602]: If we wanted to optimize this, we could potentially do 1 network call with `getMultipleAccounts` and then compute based off of this state. Not important though

File: rust/Cargo.toml [hunk 2]
```diff
@@ -31,3 +31,4 @@ rust_decimal = "1.26.1"
 rust_decimal_macros = "1.26"
 itertools = "0.10.5"
 bytemuck = "1.11.0"
+phoenix-seat-manager = { path = "../../phoenix-seat-manager" }
```
Comment (jarry-xiao) [inline rust/Cargo.toml:34]: Let's not merge this into the master just yet

**Review by jarry-xiao — COMMENTED** [review]
Changes look good, but we should hold off on merging until PSM get open sourced and published

**Comment by throwbackjams** [comment]
Closing this in favor of https://github.com/Ellipsis-Labs/phoenix-sdk/pull/87 which adapts to new SDK APIs

**Pull request closed without merging**